### PR TITLE
Subscriptions: Add List.Enabled, List.Public and List.Private

### PR DESCRIPTION
**Pull request by gbjk**
## Type of change

- [ ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Review by shazbert — APPROVED** [review]
tACK.

File: exchanges/subscription/list.go [hunk 1]
```diff
@@ -113,3 +113,36 @@ func (l List) assetPairs(e iExchange) (assetPairs, error) {
...
+	for _, s := range l {
+		if s.Enabled {
+			n = append(n, s)
+		}
+	}
+	return slices.Clip(n)
+}
+
+// Private returns only Authenticated subscriptions
+func (l List) Private() List {
```
Comment (gloriousCode) [inline exchanges/subscription/list.go:129]: Why introduce the field as Authenticated and then use a different adjective? rabble rabble rabble

**Review by gloriousCode — APPROVED** [review]
tACK

File: exchanges/subscription/list.go (hunk 1 above)
Comment (gbjk) [reply exchanges/subscription/list.go:129]: I struggled with naming on this.
When I added the `Public` method, things became clearer.

My historical issue was that `Authenticated` might sound like "Subscriptions which have already been authenticated".
And in that sense, I now want to improve this comment.
Fixed c1eb4f5e1

But you're right. I think I'd like to rename the field from Authenticated to Private too.
It's just more invasive than I wanted to be right now. I was thinking I'd do it gently once this group of PRs has landed.

**Review by thrasher- — APPROVED** [review]
Some lovely helper methods. tACK!

**Pull request merged**